cmd/end2endtest/plaintext.go

```diff
@@ -0,0 +1,381 @@
+package main
```

Comment:
Many of the comments I made before can be applied on this file too. So I'm not reviewing further until all previous comments are addressed.